### ISSUE TITLE
Test suite depends on ordering of serialized json objects

### ISSUE THREAD

**Issue by sternenseemann** [ssadler/aeson-quick#3]
This is not stable between releases of `unordered-containers`: `cabal v2-test --constraint 'unordered-containers >= 0.2.16'` yields the following:

```
Running 1 test suites...
Test suite aeson-quick-test: RUNNING...
Tests
  oneKey
    get:             OK
    set:             OK
  multipleKeys
    get:             OK
    set:             FAIL
      test/Test.hs:161:
      expected: "{\"a\":2,\"b\":[1,1]}"
       but got: "{\"b\":[1,1],\"a\":2}"
      Use -p '/multipleKeys.set/' to rerun this test only.
  deepKey
    get:             OK
    set:             OK
  keyInArray
    get:             OK
    set:             OK
  complex
    get:             OK
    set:             FAIL
      test/Test.hs:161:
      expected: "{\"a\":2,\"b\":[{\"a\":[1]}]}"
       but got: "{\"b\":[{\"a\":[1]}],\"a\":2}"
      Use -p '/complex.set/' to rerun this test only.
  optionalKey
    get:             OK
  nonExistentKey
    get:             OK
    set:             OK
    setDeep:         OK
    setDeepArray:    OK
    setDeepMany:     OK
  asLens
    get:             SKIPPED OK
    getDoesNotExist: OK
    set:             FAIL
      test/Test.hs:161:
      expected: "{\"a\":2,\"b\":1}"
       but got: "{\"b\":1,\"a\":2}"
      Use -p '/asLens.set/' to rerun this test only.
  parseInvalid
    noKey:           OK
  showStructure
    all:             OK
  quotedKey
    chars:           OK

3 out of 22 tests failed (0.00s)
Test suite aeson-quick-test: FAIL
Test suite logged to: dist/test/aeson-quick-0.1.3-aeson-quick-test.log
0 of 1 test suites (0 of 1 test cases) passed.
```

**Comment by ssadler** [comment]
Hi there,

I couldn't reproduce this, I had to constrain aeson to <= 1.5.6.0 to get it to compile but then with ghc-8.6.5 and unordered-containers constrained as above the test suite still passes. Can you perhaps provide ghc version and a `cabal v2-freeze` file? Thanks.

**Comment by ssadler** [comment]
Fixed in [0.2.0](https://hackage.haskell.org/package/aeson-quick)
